src/database/migrations/0007-create-proposal_versions.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE proposal_versions (
```

Comment:
What is the advantage of this table versus adding the columns `application_form_id` and `version` to the `proposals` table?

Reply:
I think I see it (going back to the original ER diagram): the proposal outcome can relate to the proposal, not to a particular version of the proposal.

Reply:
That's one, example yes.  Also an applicant can submit more than one distinct proposal to the same opportunity which would in turn have multiple versions!